schemas/session-replay-browser-events-schema.json

```diff
@@ -0,0 +1,12 @@
+{
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏why `-events` in file names? can we remove that, to avoid confusion with RUM events?

Reply:
I agree. If Mobile SDK agrees too, I'll send an update. (cc @mariusc83 @ncreated)

Reply:
I don't see any problem with that